helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -1045,4 +1053,28 @@ public int hashCode() {
   public String getClusterName() {
     return _record.getId();
   }
+
+  public String getPlainInstanceHelixDisabledType(String instanceName) {
+    return StringProcessUtil.concatenateConfigParser(getDisabledInstances().get(instanceName))
+        .get(ClusterConfigProperty.HELIX_DISABLED_TYPE.toString());
+  }
```

Comment:
Open discussion, can this and below methods be put in the util class instead? I think this ClusterConfig class is getting bloated.

Reply:
I think a getter for properties stored in clusterConfig is OK to be put into this class. Parser or other processor should be moved to util.